src/applications/vaos/tests/new-appointment/components/VAFacilityPage/VAFacilityPageV2.multi.unit.spec.jsx

```diff
@@ -426,6 +426,12 @@ describe('VAOS integration: VA flat facility page - multiple facilities', () =>
     expect(
       await screen.findByLabelText(/Fake facility name 1/i),
     ).to.have.attribute('checked');
+
+    expect(
+      await screen.queryByText(
+        /You’ve reached the limit for appointment request/i,
+      ),
+    ).to.be.null;
```

Comment:
When returning to the page, eligibility modal shouldn't show even if they are ineligible